docs/sphinx/user/features.rst

```diff
@@ -179,8 +179,12 @@ Current development
 
 * Temporal and spatial varying MMC forcing
 
+* non-uniform roughness length, surface flux and temperature 
```

Comment:
Can you link to any other documentation in other spots of the manual in case someone wants to learn more about these features?

Reply:
I do not have a documentation for it. Do I link to the POC draft PR?

Reply:
If this isn't in this PR, then let's not add that stuff here now. Only add to the "features" what is either in the relevant PR or what is already in the code, not what is upcoming in future PRs. Does that make sense?